### PR TITLE
removed pygame.cdrom refs everywhere

### LINKED ISSUE #2323: Remove all traces of `pygame.cdrom`?
Should we? Is it even possible to get that version of pygame anymore? Certainly `pygame-ce` doesn't have a release for it. Also, pretty sure that you can't even compile `pygame-ce` with SDL1 even if you could get SDL1

### PR DESCRIPTION
**Pull request by oddbookworm**
Resolves #2323

### CODE REVIEW

**Review by Starbuck5 — APPROVED** [review]
Thanks!

File: docs/README.md [hunk 1]
```diff
@@ -66,7 +66,7 @@ in the future.
 
 ### Hidden modules
 
-Pygame still has documentation for the old cdrom and Overlay modules, which
+Pygame still has documentation for the  Overlay module, which
```
Comment (MightyJosip) [inline docs/README.md:69]: One space too much

Comment (oddbookworm) [reply docs/README.md]: Fixed

File: test/__init__.py [hunk 2]
```diff
@@ -12,7 +12,7 @@
 
 The xxxx_test submodules of the tests package are unit test suites for
 individual parts of Pygame. Each can also be run as a main program. This is
-useful if the test, such as cdrom_test, is interactive.
+useful if the test, is interactive.
```
Comment (ankith26) [inline test/__init__.py:15]: minor nit: extra comma here

Comment (oddbookworm) [reply test/__init__.py:15]: fixed

**Review by zoldalma999 — APPROVED** [review]
Looks good to me!

**Pull request merged**